### DISCUSSION TITLE
cannot use rerank (BAAI/bge-base-en-v1.5)

### DISCUSSION THREAD

**Discussion by BlazJurisic** [michaelfeil/infinity#107] [Q&A]
"message": "InternalServerError: the loaded moded cannot fullyfill `rerank`.options are {'embed'} inherited from model_class=<class 'infinity_emb.transformer.embedder.sentence_transformer.SentenceTransformerPatched'>",

**Comment by michaelfeil** [comment]
bge-base-en is not an reranker model. It does not have pytorch weights for classification / a head.

use this model for that: https://huggingface.co/BAAI/bge-reranker-base

**Reply by BlazJurisic** [reply]
I was confused because the example uses BAAI/bge-small-en-v1.5
